runtime/doc/lsp.txt

```diff
@@ -619,19 +619,44 @@ to the callback in the "data" table. The token fields are documented in
 Note: doing anything other than calling
 |vim.lsp.semantic_tokens.highlight_token()| is considered experimental.
 
-Also the following |User| |autocommand|s are provided:
+
+LspRequest                                                        *LspRequest*
```

Comment:
for completeness these LspX events should also be listed in `:help events`, even if the full docs live here.

Reply:
Wouldn't there be conflicting vimdoc tags if they were in multiple places? `|LspRequest|` should link here, not to the one in autocmd.txt if we put it in both places, I think.

Either way, maybe another PR to shore that up? `:help events` doesn't have the other LSP related autocommands there yet, either (like `LspAttach` and `LspDetach`)